packages/monorepo-tools/rewrite_history_from.sh

```diff
@@ -35,3 +52,5 @@ SUBDIRECTORY=$SUBDIRECTORY SUBDIRECTORY_SED=${SUBDIRECTORY//-/\\-} TAB=$'\t' XAR
     --tag-name-filter 'cat' \
     -- $REV_LIST_PARAMS
 
+# Moves backup tag refs to the same group with automatic backup from `git filter-branch`
```

Comment:
So you backup all tags. Next, you are removing some tags - tags that are unnecessary for an actual split of a package, finally, you restore all tags in monorepo. Am I right?

Reply:
Almost exactly!
1. Backup monorepo tags :heavy_check_mark: 
2. Remove tags that are unnecessary for split :heavy_check_mark: 
3. Restoring is not the right word, because monorepo tags are not restored, they are "restored"/moved into refs/original section, so they can be restored by git later.